requirements.txt

```diff
@@ -1,3 +1,4 @@
+setuptools
```

Comment:
Is this now necessary?

Reply:
I don't think it's necessary for the CI, but it is necessary to run the tests on python 3.12.